src/main/java/browserview/BrowserComponent.java

```diff
@@ -486,4 +486,9 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
+    public void loadTab(String currentUrl) {
+        if (isCurrentUrlIssue() && (currentUrl.endsWith("/commits") || currentUrl.endsWith("/files"))) {
+            driver.findElement(By.xpath("//a[@data-container-id='discussion_bucket']")).click();
```

Comment:
@lejolly no that would involve a page refresh for no reason. We already have visited that page before when user initially clicked on the PR issue in the p view